MusicStore.DataAccess/Realization/AdminRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MusicStore.DataAccess.Interfaces;
+
+namespace MusicStore.DataAccess.Realization
+{
+    public class AdminRepository: IAdminRepository
```

Comment:
Such repository is not a classical Repository pattern.
Please, check if you can provide filtering in service